inc/user.class.php

```diff
@@ -2160,22 +2160,23 @@ function () {
          echo "<td><label for='password'>" . __('Password')."</label></td>";
          echo "<td><input id='password' type='password' name='password' value='' size='20'
                     autocomplete='new-password' onkeyup=\"return passwordCheck();\"></td>";
-         echo "<td rowspan='2'>";
          if ($CFG_GLPI["use_password_security"]) {
```

Comment:
This is still not correct on my instance:
![image](https://user-images.githubusercontent.com/33253653/82187619-f8143880-98ec-11ea-9fe6-06446a9be108.png)